src/components/faceVerification/screens/IntroScreen.js

```diff
@@ -57,51 +57,54 @@ const WalletDeletedPopupText = ({ styles }) => (
+const Intro = ({ styles, ready, onVerify, onLearnMore }) => {
+  const { fullName } = useProfile()
+  const firstName = getFirstWord(fullName)
+
+  return (
+    <Wrapper>
+      <Section style={styles.topContainer} grow>
+        <View style={styles.mainContent}>
+          <Section.Title fontWeight="bold" textTransform="none" style={styles.mainTitle}>
+            {firstName && `${firstName},`}
+            <Section.Text fontWeight="regular" textTransform="none" fontSize={24} lineHeight={30}>
+              {firstName ? `\n` : ''}
+              {'Verify you are a real\nlive person'}
+            </Section.Text>
+          </Section.Title>
+          <Section.Text fontSize={18} lineHeight={25} letterSpacing={0.18} style={styles.mainText}>
+            {t`Your image is only used to prevent the creation of duplicate accounts and will never be transferred to any third party`}
           </Section.Text>
-        </Section.Title>
-        <Section.Text fontSize={18} lineHeight={25} letterSpacing={0.18} style={styles.mainText}>
-          {t`Your image is only used to prevent the creation of duplicate accounts and will never be transferred to any third party`}
-        </Section.Text>
-        <Section.Text
-          fontWeight="bold"
-          fontSize={18}
-          lineHeight={26}
-          textDecorationLine="underline"
-          style={styles.learnMore}
-          onPress={onLearnMore}
-        >
-          {t`Learn More`}
-        </Section.Text>
-        <View style={styles.illustration}>
-          <FashionShootSVG />
+          <Section.Text
+            fontWeight="bold"
+            fontSize={18}
+            lineHeight={26}
+            textDecorationLine="underline"
+            style={styles.learnMore}
+            onPress={onLearnMore}
+          >
+            {t`Learn More`}
+          </Section.Text>
+          <View style={styles.illustration}>
+            <FashionShootSVG />
+          </View>
+          <CustomButton style={[styles.button]} onPress={onVerify} disabled={!ready}>
+            {t`OK, VERIFY ME`}
+          </CustomButton>
         </View>
-        <CustomButton style={[styles.button]} onPress={onVerify} disabled={!ready}>
-          {t`OK, VERIFY ME`}
-        </CustomButton>
-      </View>
-    </Section>
-  </Wrapper>
-)
+      </Section>
+    </Wrapper>
+  )
+}
 
 const IntroScreen = ({ styles, screenProps, navigation }) => {
-  const { fullName } = useProfile()
   const { showDialog } = useDialog()
 
-  const { firstName, isFVFlow, isFVFlowReady } = useContext(FVFlowContext)
+  const { isFVFlow, isFVFlowReady } = useContext(FVFlowContext)
   const { screenState, goToRoot, navigateTo, pop, push } = screenProps
   const isValid = get(screenState, 'isValid', false)
 
   const { faceIdentifier: enrollmentIdentifier, v1FaceIdentifier: fvSigner } = useEnrollmentIdentifier()
-  const userName = useMemo(() => (isFVFlow ? firstName : getFirstWord(fullName)), [isFVFlow, firstName, fullName])
```

Comment:
the actual fix should look like this one
```
const userName = useMemo(() => firstName ? (isFVFlow ? firstName : getFirstWord(fullName)) : '', [isFVFlow, firstName, fullName])
```

Reply:
done

Reply:
i dont think this is correct and i dont understand what was wrong with the first code.
if we are not in "fvflow" firstname will be null, this means that for regular wallet flows userName will be empty.
@L03TJ3 @johnsmith-gooddollar